src/libraries/Microsoft.PowerFx.Core/Types/DType.cs

```diff
@@ -1087,6 +1087,7 @@ public DType ToTable(ref bool fError)
 
             switch (Kind)
             {
+                case DKind.LazyTable:
                 case DKind.Table:
```

Comment:
>  case DKind.Table: [](http://example.com/codeflow?start=14&length=19)

This looks good - can we check other cases in this file (or does PAClient have other cases too?)